jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -53,6 +53,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <langapi/mode.h>
 
+#include <cbmc/bmc.h> // will go away
```

Comment:
Would you mind adding an issue or are you tracking this in some way?

Reply:
That will be the result as soon as I reach the top of this tree of PRs.